admob/src/android/admob_android.cc

```diff
@@ -127,16 +274,21 @@ InitResult Initialize(JNIEnv* env, jobject activity) {
   if (!util::FindClass(env, "com/google/android/gms/ads/internal/ClientApi") &&
       google_play_services::CheckAvailability(env, activity) !=
           google_play_services::kAvailabilityAvailable) {
-    return kInitResultFailedMissingDependency;
-  }
-
-  if (g_initialized) {
```

Comment:
I'm hesitant to allow this double-initialization thing because there may already be a Future in progress, and I just don't want to deal with reconciling them.